lib/docs/filters/sass/entries.rb

```diff
@@ -1,89 +1,53 @@
 module Docs
   class Sass
     class EntriesFilter < Docs::EntriesFilter
-      TYPES = ['CSS Extensions', 'SassScript', '@-Rules and Directives',
-        'Output Styles']
-
-      SKIP_NAMES = ['Interactive Shell', 'Data Types', 'Operations',
-        'Division and /', 'Keyword Arguments']
-
-      REPLACE_NAMES = {
-        '%foo'               => '%placeholder selector',
-        '&'                  => '& parent selector',
-        '$'                  => '$ variables',
-        '`'                  => '#{} interpolation',
-        'The !optional Flag' => '!optional'
-      }
 
       def get_name
-        'Functions'
+        at_css('#main-content > h1').content
       end
 
       def get_type
-        'Functions'
+        return 'Syntax' if slug.include?('syntax')
+        return 'Style rules' if slug.include?('style')
+        return 'At-Rules' if slug.include?('at')
+        return 'Values' if slug.include?('values')
+        return 'Operators' if slug.include?('operators')
+        return 'Command line' if slug.include?('cli')
+        return 'Modules' if slug.include?('modules')
+        'Misc'
```

Comment:
This is only a code style thing, but a cleaner way to do this is with this code:
```ruby
case slug
when /syntax/
  'Syntax'
when /style/
  'Style'
...
else
  'Misc'
end
```

Reply:
Thank you!, any suggestion to improve the quality of the code is welcome